here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/val/MockContextLoaderHandler.java

```diff
@@ -59,24 +60,28 @@ public MockContextLoaderHandler(
     this.properties = JsonSerializable.convert(eventHandler.getProperties(), XyzProperties.class);
   }
 
+  @Override
+  protected EventProcessingStrategy processingStrategyFor(IEvent event) {
+    Request<?> request = event.getRequest();
+    if (request instanceof WriteFeatures<?, ?, ?>) {
+      return PROCESS;
+    }
+    return SUCCEED_WITHOUT_PROCESSING;
```

Comment:
This handler being an intermediate handler, I recommend using SEND_UPSTREAM_WITHOUT_PROCESSING as default behaviour.
Only for ReadFeatures we were returning empty Success (which we should continue), for temporary mock reasons.